autoscheduler/frontend/src/package.json

```diff
@@ -33,5 +33,10 @@
     "typescript-eslint": "0.0.1-alpha.0",
     "webpack": "^4.41.0",
     "webpack-cli": "^3.3.9"
+  },
+  "dependencies": {
+    "@types/react-redux": "^7.1.5",
+    "react-redux": "^7.1.1",
+    "redux": "^4.0.4"
```

Comment:
These are the dependencies that are going to be needed for like production right? Not just for development, so shouldn't we also have react, typescript, etc in here? And have the dependencies only needed for development, i.e. ESLint, in `devDependencies`?

I originally put them in `devDependencies` for no particular reason, so really just wondering if we should move the necessary ones into `dependencies`

Reply:
Yes, React needs to be moved to production dependencies, or else we could run into problems building for production. However, since Typescript is a development-time tool that just generates JS, Typescript is fine where it is. Jest should also be fine in devDependencies, when that gets added.

Reply:
Done

Reply:
For the record, I moved React, ReactDOM, and PropTypes